content/en/docs/ibm/create-cluster.md

```diff
@@ -44,12 +44,22 @@ Get the Kubeconfig file:
 ibmcloud ks cluster config --cluster $CLUSTER_NAME
 ```
 
-From here on, please see [Install Kubeflow](/docs/ibm/deploy/install-kubeflow).
+From here on, please see [Install Kubeflow](/docs/ibm/deploy/install-kubeflow-on-iks).
 
 
 ## Create and setup a new cluster
 
-Follow these steps to create and setup a new [IBM Cloud Kubernetes Service(IKS) cluster:
+* Use `classic` provider if you want to try out Kubeflow
+* Use `vpc-gen2` provider if you are familiar with Cloud networking and want to deploy Kubeflow on a secure environment.
```

Comment:
```suggestion
* Use a `vpc-gen2` provider if you are familiar with IBM Cloud networking and want to deploy Kubeflow on a secure environment.
```

Reply:
"a" or "the" ? And this statement is intended to be general, adding IBM makes it specific.